### PR TITLE
Invite new member

### PR DESCRIPTION
**Pull request by sernik73**

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -98,6 +98,30 @@
     "invited": "invited",
     "deactivated": "deactivated"
   },
+  "addMember": {
+    "title": "Invite new member",
```
Comment (kirillta) [inline src/locales/en/translation.json:102]: Invite member

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -98,6 +98,30 @@
...
+    "fields": {
+      "firstName": {
+        "label": "Name"
```
Comment (kirillta) [inline src/locales/en/translation.json:105]: First name

File: src/locales/en/translation.json [hunk 3]
```diff
@@ -98,6 +98,30 @@
...
+      },
+      "lastName": {
+        "label": "Surname"
```
Comment (kirillta) [inline src/locales/en/translation.json:108]: Last name

File: src/locales/en/translation.json [hunk 4]
```diff
@@ -98,6 +98,30 @@
...
+      },
+      "email": {
+        "label": "E-mail"
```
Comment (kirillta) [inline src/locales/en/translation.json:111]: Email

File: src/locales/en/translation.json [hunk 5]
```diff
@@ -98,6 +98,30 @@
...
+      }
+    },
+    "formRequiredError": "Please, fill-in the required fields",
```
Comment (kirillta) [inline src/locales/en/translation.json:114]: Please, fill-in required fields

File: src/locales/en/translation.json [hunk 6]
```diff
@@ -98,6 +98,30 @@
...
+    "buttons": {
+      "back": "Back",
+      "submit": "Invite"
+    },
+    "success": {
+      "btnHome": "Home",
+      "message": "Member invited",
```
Comment (kirillta) [inline src/locales/en/translation.json:121]: Member has been invited

File: src/locales/en/translation.json [hunk 7]
```diff
@@ -98,6 +98,30 @@
...
+      "btnAdd": "Add yet member"
```
Comment (kirillta) [inline src/locales/en/translation.json:122]: Add another

File: src/locales/en/translation.json [hunk 8]
```diff
@@ -98,6 +98,30 @@
...
+    }
+  },
```
Comment (kirillta) [inline src/locales/en/translation.json:124]: Also

line 6: "transaction": "Transactions"
line 161: "formRequiredError": "Please, fill-in required fields"

File: src/locales/en/translation.json (hunk 1 above)
Comment (sernik73) [reply src/locales/en/translation.json]: done

File: src/locales/en/translation.json (hunk 2 above)
Comment (sernik73) [reply src/locales/en/translation.json]: done

File: src/locales/en/translation.json (hunk 3 above)
Comment (sernik73) [reply src/locales/en/translation.json]: done

File: src/locales/en/translation.json (hunk 4 above)
Comment (sernik73) [reply src/locales/en/translation.json]: done

File: src/locales/en/translation.json (hunk 5 above)
Comment (sernik73) [reply src/locales/en/translation.json]: done

File: src/locales/en/translation.json (hunk 6 above)
Comment (sernik73) [reply src/locales/en/translation.json]: done

File: src/locales/en/translation.json (hunk 7 above)
Comment (sernik73) [reply src/locales/en/translation.json]: done

File: src/locales/en/translation.json (hunk 8 above)
Comment (sernik73) [reply src/locales/en/translation.json:125]: done

**Review by kirillta — APPROVED** [review]

**Pull request merged**